README.md

```diff
@@ -1,7 +1,11 @@
 # OpenTelemetry Operations Exporters for JavaScript
 
+The packages in this repository support Node.js version 10 and above.
```

Comment:
Why not 9?
Based on the Circle CI config above, we never checked support for Node 9 and 11 (not sure what are the reasons for that).
Do we need to mention not supporting Node 11 then? Or do we need to add a missing coverage to Circle CI config?

Reply:
Even numbered releases are LTS, so node9 and node11 have been deprecated for a while ([npm releases info](https://nodejs.org/en/about/releases/)):
> Major Node.js versions enter Current release status for six months, which gives library authors time to add support for them. After six months, odd-numbered releases (9, 11, etc.) become unsupported, and even-numbered releases (10, 12, etc.) move to Active LTS status and are ready for general use. LTS release status is "long-term support", which typically guarantees that critical bugs will be fixed for a total of 30 months. Production applications should only use Active LTS or Maintenance LTS releases.

![image](https://raw.githubusercontent.com/nodejs/Release/master/schedule.svg?sanitize=true)

Reply:
Updated :)

Reply:
Thanks!